src/whylogs/app/session.py

```diff
@@ -259,10 +262,8 @@ def remove_logger(self, dataset_name):
 
         """
         if self._loggers.get(dataset_name) is None:
-            print(
-                "WARNING: logger {} is not present in the current Session".format(
-                    dataset_name
-                )
+            py_logger.info(
```

Comment:
This may be relevant.  Copied from [stackoverflow](https://stackoverflow.com/questions/9595009/warnings-warn-vs-logging-warning):

> warnings.warn() in library code if the issue is avoidable and the client application should be modified to eliminate the warning
> 
> logging.warning() if there is nothing the client application can do about the situation, but the event should still be noted